modules/programs/neovim.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.neovim;
+in
+{
+  options = {
+    programs.neovim = {
+      enable = mkEnableOption "Neovim";
+
+      withPython = mkOption {
+        type = types.nullOr types.bool;
+        default = true;
+        description = ''
+          Enable python2 provider. Set to true to use python2-based plugins
+        '';
+      };
+
+      extraPythonPackages = mkOption {
+        type = types.listOf types.package;
+        default = [ ];
+        example = [ ];
```

Comment:
The example should never be the same as the default value since then the reader of the documentation will just see twice that `[]` is a valid value. So the example should be either removed or non-empty (same goes for the Python 3 option below).